[2023-11-14 08:10:15] ALERT    [__main__] [custom_logger.py:alert:49] alert message
[2023-11-14 08:10:15] USER     [__main__] [custom_logger.py:user:52] user message
[2023-11-14 08:10:57] ALERT    [__main__] [custom_logger.py:alert:49] alert message
[2023-11-14 08:10:57] USER     [__main__] [custom_logger.py:user:52] user message
[2023-11-14 08:13:37] DEBUG    [__main__] [custom_logger.py:<module>:55] debug message
[2023-11-14 08:13:37] INFO     [__main__] [custom_logger.py:<module>:56] info message
[2023-11-14 08:13:37] WARNING  [__main__] [custom_logger.py:<module>:57] warning message
[2023-11-14 08:13:37] ERROR    [__main__] [custom_logger.py:<module>:58] error message
[2023-11-14 08:13:37] CRITICAL [__main__] [custom_logger.py:<module>:59] critical message
[2023-11-14 08:13:37] ALERT    [__main__] [custom_logger.py:alert:45] alert message
[2023-11-14 08:13:37] USER     [__main__] [custom_logger.py:user:48] user message
[2023-11-14 08:14:12] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message
[2023-11-14 08:14:12] INFO     [helpers.custom_logger] [app.py:<module>:9] info message
[2023-11-14 08:14:12] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message
[2023-11-14 08:14:12] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message
[2023-11-14 08:14:12] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message
[2023-11-14 08:14:12] ALERT    [helpers.custom_logger] [custom_logger.py:alert:45] alert message
[2023-11-14 08:14:12] USER     [helpers.custom_logger] [custom_logger.py:user:48] user message
[2023-11-14 08:15:13] DEBUG    [__main__] [custom_logger.py:<module>:55] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] INFO     [__main__] [custom_logger.py:<module>:56] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] WARNING  [__main__] [custom_logger.py:<module>:57] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] ERROR    [__main__] [custom_logger.py:<module>:58] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] CRITICAL [__main__] [custom_logger.py:<module>:59] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] ALERT    [__main__] [custom_logger.py:alert:45] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:15:13] USER     [__main__] [custom_logger.py:user:48] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py
[2023-11-14 08:16:35] DEBUG    [__main__] [custom_logger.py:<module>:55] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 55
[2023-11-14 08:16:35] INFO     [__main__] [custom_logger.py:<module>:56] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 56
[2023-11-14 08:16:35] WARNING  [__main__] [custom_logger.py:<module>:57] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 57
[2023-11-14 08:16:35] ERROR    [__main__] [custom_logger.py:<module>:58] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 58
[2023-11-14 08:16:35] CRITICAL [__main__] [custom_logger.py:<module>:59] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 59
[2023-11-14 08:16:35] ALERT    [__main__] [custom_logger.py:alert:45] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 45
[2023-11-14 08:16:35] USER     [__main__] [custom_logger.py:user:48] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 48
[2023-11-14 08:16:46] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8
[2023-11-14 08:16:46] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9
[2023-11-14 08:16:46] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10
[2023-11-14 08:16:46] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11
[2023-11-14 08:16:46] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12
[2023-11-14 08:16:46] ALERT    [helpers.custom_logger] [custom_logger.py:alert:45] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 45
[2023-11-14 08:16:46] USER     [helpers.custom_logger] [custom_logger.py:user:48] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 48
[2023-11-14 08:17:38] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py
[2023-11-14 08:17:38] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py
[2023-11-14 08:17:38] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py
[2023-11-14 08:17:38] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py
[2023-11-14 08:17:38] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py
[2023-11-14 08:17:38] ALERT    [helpers.custom_logger] [custom_logger.py:alert:45] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 45 alert custom_logger.py
[2023-11-14 08:17:38] USER     [helpers.custom_logger] [custom_logger.py:user:48] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 48 user custom_logger.py
[2023-11-14 08:24:40] DEBUG    [__main__] [custom_logger.py:<module>:58] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 58 <module> custom_logger.py
[2023-11-14 08:24:40] INFO     [__main__] [custom_logger.py:<module>:59] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 59 <module> custom_logger.py
[2023-11-14 08:24:40] WARNING  [__main__] [custom_logger.py:<module>:60] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 60 <module> custom_logger.py
[2023-11-14 08:24:40] ERROR    [__main__] [custom_logger.py:<module>:61] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 61 <module> custom_logger.py
[2023-11-14 08:24:40] CRITICAL [__main__] [custom_logger.py:<module>:62] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 62 <module> custom_logger.py
[2023-11-14 08:24:40] ALERT    [__main__] [custom_logger.py:alert:46] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 46 alert custom_logger.py
[2023-11-14 08:24:40] USER     [__main__] [custom_logger.py:user:50] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 50 user custom_logger.py
[2023-11-14 08:24:46] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py
[2023-11-14 08:24:46] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py
[2023-11-14 08:24:46] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py
[2023-11-14 08:24:46] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py
[2023-11-14 08:24:46] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py
[2023-11-14 08:24:46] ALERT    [helpers.custom_logger] [custom_logger.py:alert:46] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 46 alert custom_logger.py
[2023-11-14 08:24:46] USER     [helpers.custom_logger] [custom_logger.py:user:50] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 50 user custom_logger.py
[2023-11-14 08:29:34] DEBUG    [__main__] [custom_logger.py:<module>:58] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 58 <module> custom_logger.py
[2023-11-14 08:29:34] INFO     [__main__] [custom_logger.py:<module>:59] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 59 <module> custom_logger.py
[2023-11-14 08:29:34] WARNING  [__main__] [custom_logger.py:<module>:60] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 60 <module> custom_logger.py
[2023-11-14 08:29:34] ERROR    [__main__] [custom_logger.py:<module>:61] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 61 <module> custom_logger.py
[2023-11-14 08:29:34] CRITICAL [__main__] [custom_logger.py:<module>:62] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 62 <module> custom_logger.py
[2023-11-14 08:29:34] ALERT    [__main__] [custom_logger.py:<module>:63] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 63 <module> custom_logger.py
[2023-11-14 08:29:34] USER     [__main__] [custom_logger.py:<module>:64] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 64 <module> custom_logger.py
[2023-11-14 08:29:40] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py
[2023-11-14 08:29:40] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py
[2023-11-14 08:29:40] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py
[2023-11-14 08:29:40] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py
[2023-11-14 08:29:40] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py
[2023-11-14 08:29:40] ALERT    [helpers.custom_logger] [app.py:<module>:13] alert message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 13 <module> app.py
[2023-11-14 08:29:40] USER     [helpers.custom_logger] [app.py:<module>:14] user message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 14 <module> app.py
[2023-11-14 08:30:49] DEBUG    [__main__] [custom_logger.py:<module>:68] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 68 <module> custom_logger.py
[2023-11-14 08:30:49] INFO     [__main__] [custom_logger.py:<module>:69] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 69 <module> custom_logger.py
[2023-11-14 08:30:49] WARNING  [__main__] [custom_logger.py:<module>:70] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 70 <module> custom_logger.py
[2023-11-14 08:30:49] ERROR    [__main__] [custom_logger.py:<module>:71] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 71 <module> custom_logger.py
[2023-11-14 08:30:49] CRITICAL [__main__] [custom_logger.py:<module>:72] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 72 <module> custom_logger.py
[2023-11-14 08:30:49] ERROR    [__main__] [custom_logger.py:<module>:73] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 73 <module> custom_logger.py
NoneType: None
[2023-11-14 08:30:49] ALERT    [__main__] [custom_logger.py:<module>:74] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 74 <module> custom_logger.py
[2023-11-14 08:30:49] USER     [__main__] [custom_logger.py:<module>:75] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 75 <module> custom_logger.py
[2023-11-14 08:31:05] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py
[2023-11-14 08:31:05] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py
[2023-11-14 08:31:05] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py
[2023-11-14 08:31:05] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py
[2023-11-14 08:31:05] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py
[2023-11-14 08:31:05] ALERT    [helpers.custom_logger] [app.py:<module>:13] alert message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 13 <module> app.py
[2023-11-14 08:31:05] USER     [helpers.custom_logger] [app.py:<module>:14] user message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 14 <module> app.py
[2023-11-14 08:32:58] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py
[2023-11-14 08:32:58] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py
[2023-11-14 08:32:58] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py
[2023-11-14 08:32:58] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py
[2023-11-14 08:32:58] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py
[2023-11-14 08:32:58] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py
NoneType: None
[2023-11-14 08:32:58] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py
[2023-11-14 08:32:58] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py
[2023-11-14 08:32:59] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py
[2023-11-14 08:32:59] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py
[2023-11-14 08:32:59] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py
[2023-11-14 08:32:59] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py
[2023-11-14 08:32:59] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py
[2023-11-14 08:32:59] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py
NoneType: None
[2023-11-14 08:32:59] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py
[2023-11-14 08:32:59] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py
[2023-11-14 08:49:16] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:49:16] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:49:16] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:50:01] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:01] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:50:25] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:25] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:50:32] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:32] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:50:57] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:50:57] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:07] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:07] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:13] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:13] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:22] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:22] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:23] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:23] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:25] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:25] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:26] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:26] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:33] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:33] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:51:34] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:51:34] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:52:05] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:05] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:52:12] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:12] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:52:52] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] ALERT    [helpers.custom_logger] [app.py:<module>:13] alert message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 13 <module> app.py PolyLoad\app.py
[2023-11-14 08:52:52] USER     [helpers.custom_logger] [app.py:<module>:14] user message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 14 <module> app.py PolyLoad\app.py
[2023-11-14 08:54:33] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:54:33] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:54:33] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:56:06] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:06] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:56:52] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:56:52] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:57:49] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:57:49] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:58:17] DEBUG    [helpers.custom_logger] [app.py:<module>:8] debug message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 8 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] INFO     [helpers.custom_logger] [app.py:<module>:9] info message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 9 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] WARNING  [helpers.custom_logger] [app.py:<module>:10] warning message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 10 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] ERROR    [helpers.custom_logger] [app.py:<module>:11] error message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 11 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] CRITICAL [helpers.custom_logger] [app.py:<module>:12] critical message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 12 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] ALERT    [helpers.custom_logger] [app.py:<module>:13] alert message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 13 <module> app.py PolyLoad\app.py
[2023-11-14 08:58:17] USER     [helpers.custom_logger] [app.py:<module>:14] user message app c:\Users\Dima\Desktop\Github\PolyLoad\app.py 14 <module> app.py PolyLoad\app.py
[2023-11-14 08:59:07] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:59:07] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:07] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:59:36] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:36] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 08:59:52] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 08:59:52] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] DEBUG    [__main__] [custom_logger.py:<module>:85] debug message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 85 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] INFO     [__main__] [custom_logger.py:<module>:86] info message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 86 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] WARNING  [__main__] [custom_logger.py:<module>:87] warning message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 87 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] ERROR    [__main__] [custom_logger.py:<module>:88] error message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 88 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] CRITICAL [__main__] [custom_logger.py:<module>:89] critical message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 89 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] ERROR    [__main__] [custom_logger.py:<module>:90] exception message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 90 <module> custom_logger.py helpers\custom_logger.py
NoneType: None
[2023-11-14 09:01:44] ALERT    [__main__] [custom_logger.py:<module>:91] alert message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 91 <module> custom_logger.py helpers\custom_logger.py
[2023-11-14 09:01:44] USER     [__main__] [custom_logger.py:<module>:92] user message custom_logger c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py 92 <module> custom_logger.py helpers\custom_logger.py
DEBUG 2023-11-14 09:37:36.382 __main__ 2376:36204
                    <module> 105
                    debug message
INFO 2023-11-14 09:37:36.382 __main__ 2376:36204
                    <module> 106
                    info message
WARNING 2023-11-14 09:37:36.383 __main__ 2376:36204
                    <module> 107
                    warning message
ERROR 2023-11-14 09:37:36.383 __main__ 2376:36204
                    <module> 108
                    error message
CRITICAL 2023-11-14 09:37:36.383 __main__ 2376:36204
                    <module> 109
                    critical message
ERROR 2023-11-14 09:37:36.383 __main__ 2376:36204
                    <module> 110
                    exception message
NoneType: None
ALERT 2023-11-14 09:37:36.384 __main__ 2376:36204
                    <module> 111
                    alert message
USER 2023-11-14 09:37:36.384 __main__ 2376:36204
                    <module> 112
                    user message
DEBUG 2023-11-14 09:37:53.398 __main__ 49220:48244
                    <module> 105
                    debug message
INFO 2023-11-14 09:37:53.399 __main__ 49220:48244
                    <module> 106
                    info message
WARNING 2023-11-14 09:37:53.399 __main__ 49220:48244
                    <module> 107
                    warning message
ERROR 2023-11-14 09:37:53.399 __main__ 49220:48244
                    <module> 108
                    error message
CRITICAL 2023-11-14 09:37:53.400 __main__ 49220:48244
                    <module> 109
                    critical message
ERROR 2023-11-14 09:37:53.400 __main__ 49220:48244
                    <module> 110
                    exception message
NoneType: None
ALERT 2023-11-14 09:37:53.400 __main__ 49220:48244
                    <module> 111
                    alert message
USER 2023-11-14 09:37:53.400 __main__ 49220:48244
                    <module> 112
                    user message
DEBUG 2023-11-14 09:38:29.652 __main__ 30244:53164 <module> 101 debug message
INFO 2023-11-14 09:38:29.652 __main__ 30244:53164 <module> 102 info message
WARNING 2023-11-14 09:38:29.652 __main__ 30244:53164 <module> 103 warning message
ERROR 2023-11-14 09:38:29.652 __main__ 30244:53164 <module> 104 error message
CRITICAL 2023-11-14 09:38:29.652 __main__ 30244:53164 <module> 105 critical message
ERROR 2023-11-14 09:38:29.653 __main__ 30244:53164 <module> 106 exception message
NoneType: None
ALERT 2023-11-14 09:38:29.653 __main__ 30244:53164 <module> 107 alert message
USER 2023-11-14 09:38:29.653 __main__ 30244:53164 <module> 108 user message
DEBUG 2023-11-14 09:38:34.853 helpers.custom_logger 49704:20000 <module> 8 debug message
INFO 2023-11-14 09:38:34.853 helpers.custom_logger 49704:20000 <module> 9 info message
WARNING 2023-11-14 09:38:34.853 helpers.custom_logger 49704:20000 <module> 10 warning message
ERROR 2023-11-14 09:38:34.854 helpers.custom_logger 49704:20000 <module> 11 error message
CRITICAL 2023-11-14 09:38:34.854 helpers.custom_logger 49704:20000 <module> 12 critical message
ALERT 2023-11-14 09:38:34.854 helpers.custom_logger 49704:20000 <module> 13 alert message
USER 2023-11-14 09:38:34.854 helpers.custom_logger 49704:20000 <module> 14 user message
DEBUG 2023-11-14 09:39:29.52 __main__ 52604:46112 <module> 103 debug message
INFO 2023-11-14 09:39:29.52 __main__ 52604:46112 <module> 104 info message
WARNING 2023-11-14 09:39:29.52 __main__ 52604:46112 <module> 105 warning message
ERROR 2023-11-14 09:39:29.53 __main__ 52604:46112 <module> 106 error message
CRITICAL 2023-11-14 09:39:29.53 __main__ 52604:46112 <module> 107 critical message
ERROR 2023-11-14 09:39:29.53 __main__ 52604:46112 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 09:39:29.53 __main__ 52604:46112 <module> 109 alert message
USER 2023-11-14 09:39:29.53 __main__ 52604:46112 <module> 110 user message
DEBUG 2023-11-14 12:49:53.280 __main__ 8344:21932 <module> 103 debug message
INFO 2023-11-14 12:49:53.280 __main__ 8344:21932 <module> 104 info message
WARNING 2023-11-14 12:49:53.280 __main__ 8344:21932 <module> 105 warning message
ERROR 2023-11-14 12:49:53.280 __main__ 8344:21932 <module> 106 error message
CRITICAL 2023-11-14 12:49:53.280 __main__ 8344:21932 <module> 107 critical message
ERROR 2023-11-14 12:49:53.281 __main__ 8344:21932 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:49:53.281 __main__ 8344:21932 <module> 109 alert message
USER 2023-11-14 12:49:53.281 __main__ 8344:21932 <module> 110 user message
DEBUG 2023-11-14 12:49:54.983 __main__ 8860:55496 <module> 103 debug message
INFO 2023-11-14 12:49:54.983 __main__ 8860:55496 <module> 104 info message
WARNING 2023-11-14 12:49:54.983 __main__ 8860:55496 <module> 105 warning message
ERROR 2023-11-14 12:49:54.983 __main__ 8860:55496 <module> 106 error message
CRITICAL 2023-11-14 12:49:54.983 __main__ 8860:55496 <module> 107 critical message
ERROR 2023-11-14 12:49:54.984 __main__ 8860:55496 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:49:54.984 __main__ 8860:55496 <module> 109 alert message
USER 2023-11-14 12:49:54.984 __main__ 8860:55496 <module> 110 user message
DEBUG 2023-11-14 12:52:53.347 __main__ 40716:27168 <module> 103 debug message
INFO 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 104 info message
WARNING 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 105 warning message
ERROR 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 106 error message
CRITICAL 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 107 critical message
ERROR 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 109 alert message
USER 2023-11-14 12:52:53.348 __main__ 40716:27168 <module> 110 user message
DEBUG 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 103 debug message
INFO 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 104 info message
WARNING 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 105 warning message
ERROR 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 106 error message
CRITICAL 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 107 critical message
ERROR 2023-11-14 12:52:54.779 __main__ 50192:8916 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:52:54.780 __main__ 50192:8916 <module> 109 alert message
USER 2023-11-14 12:52:54.780 __main__ 50192:8916 <module> 110 user message
DEBUG 2023-11-14 12:56:02.256 __main__ 17828:18596 <module> 103 debug message
INFO 2023-11-14 12:56:02.256 __main__ 17828:18596 <module> 104 info message
WARNING 2023-11-14 12:56:02.256 __main__ 17828:18596 <module> 105 warning message
ERROR 2023-11-14 12:56:02.256 __main__ 17828:18596 <module> 106 error message
CRITICAL 2023-11-14 12:56:02.257 __main__ 17828:18596 <module> 107 critical message
ERROR 2023-11-14 12:56:02.257 __main__ 17828:18596 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:02.257 __main__ 17828:18596 <module> 109 alert message
USER 2023-11-14 12:56:02.257 __main__ 17828:18596 <module> 110 user message
DEBUG 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 103 debug message
INFO 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 104 info message
WARNING 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 105 warning message
ERROR 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 106 error message
CRITICAL 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 107 critical message
ERROR 2023-11-14 12:56:03.260 __main__ 6388:22952 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:03.261 __main__ 6388:22952 <module> 109 alert message
USER 2023-11-14 12:56:03.261 __main__ 6388:22952 <module> 110 user message
DEBUG 2023-11-14 12:56:03.728 __main__ 12672:1444 <module> 103 debug message
INFO 2023-11-14 12:56:03.728 __main__ 12672:1444 <module> 104 info message
WARNING 2023-11-14 12:56:03.728 __main__ 12672:1444 <module> 105 warning message
ERROR 2023-11-14 12:56:03.729 __main__ 12672:1444 <module> 106 error message
CRITICAL 2023-11-14 12:56:03.729 __main__ 12672:1444 <module> 107 critical message
ERROR 2023-11-14 12:56:03.729 __main__ 12672:1444 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:03.729 __main__ 12672:1444 <module> 109 alert message
USER 2023-11-14 12:56:03.729 __main__ 12672:1444 <module> 110 user message
DEBUG 2023-11-14 12:56:04.86 __main__ 1584:29016 <module> 103 debug message
INFO 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 104 info message
WARNING 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 105 warning message
ERROR 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 106 error message
CRITICAL 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 107 critical message
ERROR 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 109 alert message
USER 2023-11-14 12:56:04.87 __main__ 1584:29016 <module> 110 user message
DEBUG 2023-11-14 12:56:04.985 __main__ 18668:21628 <module> 103 debug message
INFO 2023-11-14 12:56:04.985 __main__ 18668:21628 <module> 104 info message
WARNING 2023-11-14 12:56:04.985 __main__ 18668:21628 <module> 105 warning message
ERROR 2023-11-14 12:56:04.985 __main__ 18668:21628 <module> 106 error message
CRITICAL 2023-11-14 12:56:04.985 __main__ 18668:21628 <module> 107 critical message
ERROR 2023-11-14 12:56:04.986 __main__ 18668:21628 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:04.986 __main__ 18668:21628 <module> 109 alert message
USER 2023-11-14 12:56:04.986 __main__ 18668:21628 <module> 110 user message
DEBUG 2023-11-14 12:56:05.436 __main__ 37460:53840 <module> 103 debug message
INFO 2023-11-14 12:56:05.436 __main__ 37460:53840 <module> 104 info message
WARNING 2023-11-14 12:56:05.436 __main__ 37460:53840 <module> 105 warning message
ERROR 2023-11-14 12:56:05.437 __main__ 37460:53840 <module> 106 error message
CRITICAL 2023-11-14 12:56:05.437 __main__ 37460:53840 <module> 107 critical message
ERROR 2023-11-14 12:56:05.437 __main__ 37460:53840 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:05.437 __main__ 37460:53840 <module> 109 alert message
USER 2023-11-14 12:56:05.437 __main__ 37460:53840 <module> 110 user message
DEBUG 2023-11-14 12:56:05.800 __main__ 30328:47768 <module> 103 debug message
INFO 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 104 info message
WARNING 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 105 warning message
ERROR 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 106 error message
CRITICAL 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 107 critical message
ERROR 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 109 alert message
USER 2023-11-14 12:56:05.801 __main__ 30328:47768 <module> 110 user message
DEBUG 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 103 debug message
INFO 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 104 info message
WARNING 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 105 warning message
ERROR 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 107 critical message
ERROR 2023-11-14 12:56:06.138 __main__ 44236:42548 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.139 __main__ 44236:42548 <module> 109 alert message
USER 2023-11-14 12:56:06.139 __main__ 44236:42548 <module> 110 user message
DEBUG 2023-11-14 12:56:06.374 __main__ 17904:3252 <module> 103 debug message
INFO 2023-11-14 12:56:06.374 __main__ 17904:3252 <module> 104 info message
WARNING 2023-11-14 12:56:06.374 __main__ 17904:3252 <module> 105 warning message
ERROR 2023-11-14 12:56:06.374 __main__ 17904:3252 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.374 __main__ 17904:3252 <module> 107 critical message
ERROR 2023-11-14 12:56:06.375 __main__ 17904:3252 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.375 __main__ 17904:3252 <module> 109 alert message
USER 2023-11-14 12:56:06.375 __main__ 17904:3252 <module> 110 user message
DEBUG 2023-11-14 12:56:06.531 __main__ 55480:29420 <module> 103 debug message
INFO 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 104 info message
WARNING 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 105 warning message
ERROR 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 107 critical message
ERROR 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 109 alert message
USER 2023-11-14 12:56:06.532 __main__ 55480:29420 <module> 110 user message
DEBUG 2023-11-14 12:56:06.677 __main__ 19264:39072 <module> 103 debug message
INFO 2023-11-14 12:56:06.677 __main__ 19264:39072 <module> 104 info message
WARNING 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 105 warning message
ERROR 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 107 critical message
ERROR 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 109 alert message
USER 2023-11-14 12:56:06.678 __main__ 19264:39072 <module> 110 user message
DEBUG 2023-11-14 12:56:06.827 __main__ 22460:6504 <module> 103 debug message
INFO 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 104 info message
WARNING 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 105 warning message
ERROR 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 107 critical message
ERROR 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.828 __main__ 22460:6504 <module> 109 alert message
USER 2023-11-14 12:56:06.829 __main__ 22460:6504 <module> 110 user message
DEBUG 2023-11-14 12:56:06.949 __main__ 40100:24772 <module> 103 debug message
INFO 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 104 info message
WARNING 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 105 warning message
ERROR 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 106 error message
CRITICAL 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 107 critical message
ERROR 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:06.950 __main__ 40100:24772 <module> 109 alert message
USER 2023-11-14 12:56:06.951 __main__ 40100:24772 <module> 110 user message
DEBUG 2023-11-14 12:56:07.91 __main__ 13928:29116 <module> 103 debug message
INFO 2023-11-14 12:56:07.92 __main__ 13928:29116 <module> 104 info message
WARNING 2023-11-14 12:56:07.92 __main__ 13928:29116 <module> 105 warning message
ERROR 2023-11-14 12:56:07.92 __main__ 13928:29116 <module> 106 error message
CRITICAL 2023-11-14 12:56:07.92 __main__ 13928:29116 <module> 107 critical message
ERROR 2023-11-14 12:56:07.92 __main__ 13928:29116 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 12:56:07.93 __main__ 13928:29116 <module> 109 alert message
USER 2023-11-14 12:56:07.93 __main__ 13928:29116 <module> 110 user message
DEBUG 2023-11-14 13:39:23.17 __main__ 4908:10672 <module> 103 debug message
INFO 2023-11-14 13:39:23.17 __main__ 4908:10672 <module> 104 info message
WARNING 2023-11-14 13:39:23.17 __main__ 4908:10672 <module> 105 warning message
ERROR 2023-11-14 13:39:23.17 __main__ 4908:10672 <module> 106 error message
CRITICAL 2023-11-14 13:39:23.18 __main__ 4908:10672 <module> 107 critical message
ERROR 2023-11-14 13:39:23.18 __main__ 4908:10672 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:39:23.18 __main__ 4908:10672 <module> 109 alert message
USER 2023-11-14 13:39:23.18 __main__ 4908:10672 <module> 110 user message
DEBUG 2023-11-14 13:42:16.652 __main__ 8604:8076 <module> 103 debug message
INFO 2023-11-14 13:42:16.652 __main__ 8604:8076 <module> 104 info message
WARNING 2023-11-14 13:42:16.652 __main__ 8604:8076 <module> 105 warning message
ERROR 2023-11-14 13:42:16.653 __main__ 8604:8076 <module> 106 error message
CRITICAL 2023-11-14 13:42:16.653 __main__ 8604:8076 <module> 107 critical message
ERROR 2023-11-14 13:42:16.653 __main__ 8604:8076 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:16.653 __main__ 8604:8076 <module> 109 alert message
USER 2023-11-14 13:42:16.653 __main__ 8604:8076 <module> 110 user message
DEBUG 2023-11-14 13:42:17.621 __main__ 2584:41176 <module> 103 debug message
INFO 2023-11-14 13:42:17.621 __main__ 2584:41176 <module> 104 info message
WARNING 2023-11-14 13:42:17.621 __main__ 2584:41176 <module> 105 warning message
ERROR 2023-11-14 13:42:17.622 __main__ 2584:41176 <module> 106 error message
CRITICAL 2023-11-14 13:42:17.622 __main__ 2584:41176 <module> 107 critical message
ERROR 2023-11-14 13:42:17.622 __main__ 2584:41176 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:17.622 __main__ 2584:41176 <module> 109 alert message
USER 2023-11-14 13:42:17.622 __main__ 2584:41176 <module> 110 user message
DEBUG 2023-11-14 13:42:31.812 __main__ 10848:43388 <module> 103 debug message
INFO 2023-11-14 13:42:31.812 __main__ 10848:43388 <module> 104 info message
WARNING 2023-11-14 13:42:31.812 __main__ 10848:43388 <module> 105 warning message
ERROR 2023-11-14 13:42:31.813 __main__ 10848:43388 <module> 106 error message
CRITICAL 2023-11-14 13:42:31.813 __main__ 10848:43388 <module> 107 critical message
ERROR 2023-11-14 13:42:31.813 __main__ 10848:43388 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:31.813 __main__ 10848:43388 <module> 109 alert message
USER 2023-11-14 13:42:31.813 __main__ 10848:43388 <module> 110 user message
DEBUG 2023-11-14 13:42:41.893 __main__ 6420:53612 <module> 103 debug message
INFO 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 104 info message
WARNING 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 105 warning message
ERROR 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 106 error message
CRITICAL 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 107 critical message
ERROR 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:41.894 __main__ 6420:53612 <module> 109 alert message
USER 2023-11-14 13:42:41.895 __main__ 6420:53612 <module> 110 user message
DEBUG 2023-11-14 13:42:49.611 __main__ 22076:24448 <module> 103 debug message
INFO 2023-11-14 13:42:49.611 __main__ 22076:24448 <module> 104 info message
WARNING 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 105 warning message
ERROR 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 106 error message
CRITICAL 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 107 critical message
ERROR 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 109 alert message
USER 2023-11-14 13:42:49.612 __main__ 22076:24448 <module> 110 user message
DEBUG 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 103 debug message
INFO 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 104 info message
WARNING 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 105 warning message
ERROR 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 106 error message
CRITICAL 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 107 critical message
ERROR 2023-11-14 13:42:59.808 __main__ 15448:15112 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:42:59.809 __main__ 15448:15112 <module> 109 alert message
USER 2023-11-14 13:42:59.809 __main__ 15448:15112 <module> 110 user message
DEBUG 2023-11-14 13:43:08.752 __main__ 24168:29708 <module> 103 debug message
INFO 2023-11-14 13:43:08.752 __main__ 24168:29708 <module> 104 info message
WARNING 2023-11-14 13:43:08.752 __main__ 24168:29708 <module> 105 warning message
ERROR 2023-11-14 13:43:08.752 __main__ 24168:29708 <module> 106 error message
CRITICAL 2023-11-14 13:43:08.752 __main__ 24168:29708 <module> 107 critical message
ERROR 2023-11-14 13:43:08.753 __main__ 24168:29708 <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:43:08.753 __main__ 24168:29708 <module> 109 alert message
USER 2023-11-14 13:43:08.753 __main__ 24168:29708 <module> 110 user message
DEBUG    2023-11-14 13:43:26.747 __main__ 38232:25444 <module> 103 debug message
INFO    2023-11-14 13:43:26.748 __main__ 38232:25444 <module> 104 info message
WARNING    2023-11-14 13:43:26.748 __main__ 38232:25444 <module> 105 warning message
ERROR    2023-11-14 13:43:26.748 __main__ 38232:25444 <module> 106 error message
CRITICAL    2023-11-14 13:43:26.748 __main__ 38232:25444 <module> 107 critical message
ERROR    2023-11-14 13:43:26.748 __main__ 38232:25444 <module> 108 exception message
NoneType: None
ALERT    2023-11-14 13:43:26.749 __main__ 38232:25444 <module> 109 alert message
USER    2023-11-14 13:43:26.749 __main__ 38232:25444 <module> 110 user message
DEBUG 2023-11-14 13:45:14.53 __main__ |9816:17772| <module> 103 debug message
INFO 2023-11-14 13:45:14.53 __main__ |9816:17772| <module> 104 info message
WARNING 2023-11-14 13:45:14.53 __main__ |9816:17772| <module> 105 warning message
ERROR 2023-11-14 13:45:14.53 __main__ |9816:17772| <module> 106 error message
CRITICAL 2023-11-14 13:45:14.54 __main__ |9816:17772| <module> 107 critical message
ERROR 2023-11-14 13:45:14.54 __main__ |9816:17772| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:45:14.54 __main__ |9816:17772| <module> 109 alert message
USER 2023-11-14 13:45:14.54 __main__ |9816:17772| <module> 110 user message
DEBUG 2023-11-14 13:45:18.118 helpers.custom_logger |24844:32916| <module> 8 debug message
INFO 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 9 info message
WARNING 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 10 warning message
ERROR 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 11 error message
CRITICAL 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 12 critical message
ALERT 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 13 alert message
USER 2023-11-14 13:45:18.119 helpers.custom_logger |24844:32916| <module> 14 user message
DEBUG 2023-11-14 13:45:46.903 helpers.custom_logger |54428:12540| main 7 debug message
INFO 2023-11-14 13:45:46.903 helpers.custom_logger |54428:12540| main 8 info message
WARNING 2023-11-14 13:45:46.903 helpers.custom_logger |54428:12540| main 9 warning message
ERROR 2023-11-14 13:45:46.904 helpers.custom_logger |54428:12540| main 10 error message
CRITICAL 2023-11-14 13:45:46.904 helpers.custom_logger |54428:12540| main 11 critical message
ALERT 2023-11-14 13:45:46.904 helpers.custom_logger |54428:12540| main 12 alert message
USER 2023-11-14 13:45:46.904 helpers.custom_logger |54428:12540| main 13 user message
DEBUG 2023-11-14 13:46:37.268 __main__ |8:51336| <module> 103 debug message
INFO 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 104 info message
WARNING 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 105 warning message
ERROR 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 106 error message
CRITICAL 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 107 critical message
ERROR 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 109 alert message
USER 2023-11-14 13:46:37.269 __main__ |8:51336| <module> 110 user message
DEBUG 2023-11-14 13:46:48.212 __main__ |24608:43092| <module> 103 debug message
INFO 2023-11-14 13:46:48.213 __main__ |24608:43092| <module> 104 info message
WARNING 2023-11-14 13:46:48.213 __main__ |24608:43092| <module> 105 warning message
ERROR 2023-11-14 13:46:48.213 __main__ |24608:43092| <module> 106 error message
CRITICAL 2023-11-14 13:46:48.213 __main__ |24608:43092| <module> 107 critical message
ERROR 2023-11-14 13:46:48.213 __main__ |24608:43092| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:46:48.214 __main__ |24608:43092| <module> 109 alert message
USER 2023-11-14 13:46:48.214 __main__ |24608:43092| <module> 110 user message
DEBUG 2023-11-14 13:47:26.878 __main__ |54400:44496| <module> 103 debug message
INFO 2023-11-14 13:47:26.878 __main__ |54400:44496| <module> 104 info message
WARNING 2023-11-14 13:47:26.878 __main__ |54400:44496| <module> 105 warning message
ERROR 2023-11-14 13:47:26.878 __main__ |54400:44496| <module> 106 error message
CRITICAL 2023-11-14 13:47:26.878 __main__ |54400:44496| <module> 107 critical message
ERROR 2023-11-14 13:47:26.879 __main__ |54400:44496| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:47:26.879 __main__ |54400:44496| <module> 109 alert message
USER 2023-11-14 13:47:26.879 __main__ |54400:44496| <module> 110 user message
DEBUG 2023-11-14 13:47:50.577 __main__ |9536:1260| <module> 103 debug message
INFO 2023-11-14 13:47:50.577 __main__ |9536:1260| <module> 104 info message
WARNING 2023-11-14 13:47:50.578 __main__ |9536:1260| <module> 105 warning message
ERROR 2023-11-14 13:47:50.578 __main__ |9536:1260| <module> 106 error message
CRITICAL 2023-11-14 13:47:50.578 __main__ |9536:1260| <module> 107 critical message
ERROR 2023-11-14 13:47:50.578 __main__ |9536:1260| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:47:50.578 __main__ |9536:1260| <module> 109 alert message
USER 2023-11-14 13:47:50.579 __main__ |9536:1260| <module> 110 user message
DEBUG 2023-11-14 13:48:04.658 __main__ |51944:20992| <module> 103 debug message
INFO 2023-11-14 13:48:04.659 __main__ |51944:20992| <module> 104 info message
WARNING 2023-11-14 13:48:04.659 __main__ |51944:20992| <module> 105 warning message
ERROR 2023-11-14 13:48:04.659 __main__ |51944:20992| <module> 106 error message
CRITICAL 2023-11-14 13:48:04.659 __main__ |51944:20992| <module> 107 critical message
ERROR 2023-11-14 13:48:04.660 __main__ |51944:20992| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:48:04.660 __main__ |51944:20992| <module> 109 alert message
USER 2023-11-14 13:48:04.660 __main__ |51944:20992| <module> 110 user message
DEBUG 2023-11-14 13:48:30.738 __main__ |11732:29028| <module> 103 debug message
INFO 2023-11-14 13:48:30.739 __main__ |11732:29028| <module> 104 info message
WARNING 2023-11-14 13:48:30.739 __main__ |11732:29028| <module> 105 warning message
ERROR 2023-11-14 13:48:30.739 __main__ |11732:29028| <module> 106 error message
CRITICAL 2023-11-14 13:48:30.739 __main__ |11732:29028| <module> 107 critical message
ERROR 2023-11-14 13:48:30.739 __main__ |11732:29028| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:48:30.740 __main__ |11732:29028| <module> 109 alert message
USER 2023-11-14 13:48:30.740 __main__ |11732:29028| <module> 110 user message
DEBUG 2023-11-14 13:49:14.411 __main__ |9076:10656| <module> 103 debug message
INFO 2023-11-14 13:49:14.412 __main__ |9076:10656| <module> 104 info message
WARNING 2023-11-14 13:49:14.412 __main__ |9076:10656| <module> 105 warning message
ERROR 2023-11-14 13:49:14.412 __main__ |9076:10656| <module> 106 error message
CRITICAL 2023-11-14 13:49:14.412 __main__ |9076:10656| <module> 107 critical message
ERROR 2023-11-14 13:49:14.412 __main__ |9076:10656| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:49:14.413 __main__ |9076:10656| <module> 109 alert message
USER 2023-11-14 13:49:14.413 __main__ |9076:10656| <module> 110 user message
DEBUG 2023-11-14 13:49:29.361 __main__ |20452:13000| <module> 103 debug message
INFO 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 104 info message
WARNING 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 105 warning message
ERROR 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 106 error message
CRITICAL 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 107 critical message
ERROR 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 109 alert message
USER 2023-11-14 13:49:29.362 __main__ |20452:13000| <module> 110 user message
DEBUG 2023-11-14 13:49:36.944 __main__ |2720:42980| <module> 103 debug message
INFO 2023-11-14 13:49:36.944 __main__ |2720:42980| <module> 104 info message
WARNING 2023-11-14 13:49:36.944 __main__ |2720:42980| <module> 105 warning message
ERROR 2023-11-14 13:49:36.944 __main__ |2720:42980| <module> 106 error message
CRITICAL 2023-11-14 13:49:36.944 __main__ |2720:42980| <module> 107 critical message
ERROR 2023-11-14 13:49:36.945 __main__ |2720:42980| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:49:36.945 __main__ |2720:42980| <module> 109 alert message
USER 2023-11-14 13:49:36.945 __main__ |2720:42980| <module> 110 user message
DEBUG 2023-11-14 13:49:38.141 __main__ |47912:13316| <module> 103 debug message
INFO 2023-11-14 13:49:38.141 __main__ |47912:13316| <module> 104 info message
WARNING 2023-11-14 13:49:38.141 __main__ |47912:13316| <module> 105 warning message
ERROR 2023-11-14 13:49:38.141 __main__ |47912:13316| <module> 106 error message
CRITICAL 2023-11-14 13:49:38.141 __main__ |47912:13316| <module> 107 critical message
ERROR 2023-11-14 13:49:38.142 __main__ |47912:13316| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:49:38.142 __main__ |47912:13316| <module> 109 alert message
USER 2023-11-14 13:49:38.142 __main__ |47912:13316| <module> 110 user message
DEBUG 2023-11-14 13:50:00.70 __main__ |23532:41896| <module> 103 debug message
INFO 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 104 info message
WARNING 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 105 warning message
ERROR 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 106 error message
CRITICAL 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 107 critical message
ERROR 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 109 alert message
USER 2023-11-14 13:50:00.71 __main__ |23532:41896| <module> 110 user message
DEBUG 2023-11-14 13:51:29.528 __main__ |32904:6188| <module> 103 debug message
INFO 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 104 info message
WARNING 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 105 warning message
ERROR 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 106 error message
CRITICAL 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 107 critical message
ERROR 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 109 alert message
USER 2023-11-14 13:51:29.529 __main__ |32904:6188| <module> 110 user message
DEBUG 2023-11-14 13:51:30.504 __main__ |32264:20960| <module> 103 debug message
INFO 2023-11-14 13:51:30.504 __main__ |32264:20960| <module> 104 info message
WARNING 2023-11-14 13:51:30.504 __main__ |32264:20960| <module> 105 warning message
ERROR 2023-11-14 13:51:30.504 __main__ |32264:20960| <module> 106 error message
CRITICAL 2023-11-14 13:51:30.504 __main__ |32264:20960| <module> 107 critical message
ERROR 2023-11-14 13:51:30.505 __main__ |32264:20960| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:51:30.505 __main__ |32264:20960| <module> 109 alert message
USER 2023-11-14 13:51:30.505 __main__ |32264:20960| <module> 110 user message
DEBUG 2023-11-14 13:51:49.963 __main__ |7944:16392| <module> 103 debug message
INFO 2023-11-14 13:51:49.963 __main__ |7944:16392| <module> 104 info message
WARNING 2023-11-14 13:51:49.963 __main__ |7944:16392| <module> 105 warning message
ERROR 2023-11-14 13:51:49.964 __main__ |7944:16392| <module> 106 error message
CRITICAL 2023-11-14 13:51:49.964 __main__ |7944:16392| <module> 107 critical message
ERROR 2023-11-14 13:51:49.964 __main__ |7944:16392| <module> 108 exception message
NoneType: None
ALERT 2023-11-14 13:51:49.964 __main__ |7944:16392| <module> 109 alert message
USER 2023-11-14 13:51:49.964 __main__ |7944:16392| <module> 110 user message
DEBUG 2023-11-14 13:54:35.369 __main__ |47764:55864| <module> 110 debug message
INFO 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 111 info message
INFO 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 112 warning message
WARNING 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 113 error message
CRITICAL 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 114 critical message
WARNING 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 116 alert message
USER 2023-11-14 13:54:35.370 __main__ |47764:55864| <module> 117 user message
DEBUG 2023-11-14 13:54:36.246 __main__ |51856:24368| <module> 110 debug message
INFO 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 111 info message
INFO 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 112 warning message
WARNING 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 113 error message
CRITICAL 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 114 critical message
WARNING 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 116 alert message
USER 2023-11-14 13:54:36.247 __main__ |51856:24368| <module> 117 user message
DEBUG 2023-11-14 13:54:36.697 __main__ |48428:30220| <module> 110 debug message
INFO 2023-11-14 13:54:36.697 __main__ |48428:30220| <module> 111 info message
INFO 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 112 warning message
WARNING 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 113 error message
CRITICAL 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 114 critical message
WARNING 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 116 alert message
USER 2023-11-14 13:54:36.698 __main__ |48428:30220| <module> 117 user message
DEBUG 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 110 debug message
INFO 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 111 info message
INFO 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 112 warning message
WARNING 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 113 error message
CRITICAL 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 114 critical message
WARNING 2023-11-14 13:54:37.127 __main__ |22564:11268| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:54:37.128 __main__ |22564:11268| <module> 116 alert message
USER 2023-11-14 13:54:37.128 __main__ |22564:11268| <module> 117 user message
DEBUG 2023-11-14 13:55:08.195 __main__ |42484:39436| <module> 110 debug message
INFO 2023-11-14 13:55:08.195 __main__ |42484:39436| <module> 111 info message
INFO 2023-11-14 13:55:08.195 __main__ |42484:39436| <module> 112 warning message
WARNING 2023-11-14 13:55:08.196 __main__ |42484:39436| <module> 113 error message
CRITICAL 2023-11-14 13:55:08.196 __main__ |42484:39436| <module> 114 critical message
WARNING 2023-11-14 13:55:08.196 __main__ |42484:39436| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:55:08.196 __main__ |42484:39436| <module> 116 alert message
USER 2023-11-14 13:55:08.196 __main__ |42484:39436| <module> 117 user message
DEBUG 2023-11-14 13:55:09.128 __main__ |54596:23152| <module> 110 debug message
INFO 2023-11-14 13:55:09.128 __main__ |54596:23152| <module> 111 info message
INFO 2023-11-14 13:55:09.128 __main__ |54596:23152| <module> 112 warning message
WARNING 2023-11-14 13:55:09.128 __main__ |54596:23152| <module> 113 error message
CRITICAL 2023-11-14 13:55:09.129 __main__ |54596:23152| <module> 114 critical message
WARNING 2023-11-14 13:55:09.129 __main__ |54596:23152| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:55:09.129 __main__ |54596:23152| <module> 116 alert message
USER 2023-11-14 13:55:09.129 __main__ |54596:23152| <module> 117 user message
DEBUG 2023-11-14 13:55:20.129 __main__ |29868:28328| <module> 110 debug message
INFO 2023-11-14 13:55:20.129 __main__ |29868:28328| <module> 111 info message
INFO 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 112 warning message
WARNING 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 113 error message
CRITICAL 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 114 critical message
WARNING 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 115 exception message
NoneType: None
ALERT 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 116 alert message
USER 2023-11-14 13:55:20.130 __main__ |29868:28328| <module> 117 user message
DEBUG 2023-11-14 13:57:43.55 __main__ |35764:42756| <module> 136 debug message
INFO 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 137 info message
WARNING 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 138 warning message
ERROR 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 139 error message
CRITICAL 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 140 critical message
EXCEPTION 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 141 exception message
ALERT 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 142 alert message
USER 2023-11-14 13:57:43.56 __main__ |35764:42756| <module> 143 user message
DEBUG 2023-11-14 13:57:44.921 __main__ |30368:9588| <module> 136 debug message
INFO 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 137 info message
WARNING 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 138 warning message
ERROR 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 139 error message
CRITICAL 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 140 critical message
EXCEPTION 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 141 exception message
ALERT 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 142 alert message
USER 2023-11-14 13:57:44.922 __main__ |30368:9588| <module> 143 user message
DEBUG 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 136 debug message
INFO 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 137 info message
WARNING 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 138 warning message
ERROR 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 139 error message
CRITICAL 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 140 critical message
EXCEPTION 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 141 exception message
ALERT 2023-11-14 13:58:11.175 __main__ |29132:2840| <module> 142 alert message
USER 2023-11-14 13:58:11.176 __main__ |29132:2840| <module> 143 user message
DEBUG 2023-11-14 13:58:41.376 __main__ |29916:8616| <module> 136 debug message
INFO 2023-11-14 13:58:41.376 __main__ |29916:8616| <module> 137 info message
WARNING 2023-11-14 13:58:41.376 __main__ |29916:8616| <module> 138 warning message
ERROR 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 139 error message
CRITICAL 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 140 critical message
EXCEPTION 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 141 exception message
ALERT 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 142 alert message
USER 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 143 user message
EXCEPTION 2023-11-14 13:58:41.377 __main__ |29916:8616| <module> 148 division by zero
DEBUG 2023-11-14 13:58:42.602 __main__ |24804:51984| <module> 136 debug message
INFO 2023-11-14 13:58:42.602 __main__ |24804:51984| <module> 137 info message
WARNING 2023-11-14 13:58:42.602 __main__ |24804:51984| <module> 138 warning message
ERROR 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 139 error message
CRITICAL 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 140 critical message
EXCEPTION 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 141 exception message
ALERT 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 142 alert message
USER 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 143 user message
EXCEPTION 2023-11-14 13:58:42.603 __main__ |24804:51984| <module> 148 division by zero
DEBUG 2023-11-14 13:58:43.443 __main__ |34436:45556| <module> 136 debug message
INFO 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 137 info message
WARNING 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 138 warning message
ERROR 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 139 error message
CRITICAL 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 140 critical message
EXCEPTION 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 141 exception message
ALERT 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 142 alert message
USER 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 143 user message
EXCEPTION 2023-11-14 13:58:43.444 __main__ |34436:45556| <module> 148 division by zero
DEBUG 2023-11-14 13:59:16.818 __main__ |25136:7744| <module> 136 debug message
INFO 2023-11-14 13:59:16.818 __main__ |25136:7744| <module> 137 info message
WARNING 2023-11-14 13:59:16.818 __main__ |25136:7744| <module> 138 warning message
ERROR 2023-11-14 13:59:16.818 __main__ |25136:7744| <module> 139 error message
CRITICAL 2023-11-14 13:59:16.819 __main__ |25136:7744| <module> 140 critical message
EXCEPTION 2023-11-14 13:59:16.819 __main__ |25136:7744| <module> 141 exception message
NoneType: None
ALERT 2023-11-14 13:59:16.819 __main__ |25136:7744| <module> 142 alert message
USER 2023-11-14 13:59:16.819 __main__ |25136:7744| <module> 143 user message
EXCEPTION 2023-11-14 13:59:16.819 __main__ |25136:7744| <module> 148 division by zero
Traceback (most recent call last):
  File "c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py", line 146, in <module>
    x = 1/0
        ~^~
ZeroDivisionError: division by zero
DEBUG 2023-11-14 13:59:21.562 __main__ |40164:18008| <module> 136 debug message
INFO 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 137 info message
WARNING 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 138 warning message
ERROR 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 139 error message
CRITICAL 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 140 critical message
EXCEPTION 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 141 exception message
NoneType: None
ALERT 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 142 alert message
USER 2023-11-14 13:59:21.563 __main__ |40164:18008| <module> 143 user message
EXCEPTION 2023-11-14 13:59:21.564 __main__ |40164:18008| <module> 148 division by zero
Traceback (most recent call last):
  File "c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py", line 146, in <module>
    x = 1/0
        ~^~
ZeroDivisionError: division by zero
DEBUG 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 136 debug message
INFO 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 137 info message
WARNING 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 138 warning message
ERROR 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 139 error message
CRITICAL 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 140 critical message
EXCEPTION 2023-11-14 13:59:22.145 __main__ |38560:41528| <module> 141 exception message
NoneType: None
ALERT 2023-11-14 13:59:22.146 __main__ |38560:41528| <module> 142 alert message
USER 2023-11-14 13:59:22.146 __main__ |38560:41528| <module> 143 user message
EXCEPTION 2023-11-14 13:59:22.146 __main__ |38560:41528| <module> 148 division by zero
Traceback (most recent call last):
  File "c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py", line 146, in <module>
    x = 1/0
        ~^~
ZeroDivisionError: division by zero
DEBUG 2023-11-14 13:59:24.97 __main__ |36020:7616| <module> 136 debug message
INFO 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 137 info message
WARNING 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 138 warning message
ERROR 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 139 error message
CRITICAL 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 140 critical message
EXCEPTION 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 141 exception message
NoneType: None
ALERT 2023-11-14 13:59:24.98 __main__ |36020:7616| <module> 142 alert message
USER 2023-11-14 13:59:24.99 __main__ |36020:7616| <module> 143 user message
EXCEPTION 2023-11-14 13:59:24.99 __main__ |36020:7616| <module> 148 division by zero
Traceback (most recent call last):
  File "c:\Users\Dima\Desktop\Github\PolyLoad\helpers\custom_logger.py", line 146, in <module>
    x = 1/0
        ~^~
ZeroDivisionError: division by zero
